packages/mds-compliance/mds-compliance-engine.ts

```diff
@@ -212,8 +212,9 @@ function processPolicy(
   devices: { [d: string]: Device }
 ): ComplianceResponse | undefined {
   if (isPolicyActive(policy)) {
-    const vehiclesToFilter: MatchedVehicle[] = []
+    let total_violations = 0
```

Comment:
This seems really side-effecty... what happens if you have a policy with both a count max rule and then a count min rule? If they should both have violations, then I think the violations in the min rule will completely overwrite the violations from the count max rule.